nav2_bringup/launch/localization_launch.py

```diff
@@ -102,6 +101,7 @@ def generate_launch_description():
     load_nodes = GroupAction(
         condition=IfCondition(PythonExpression(['not ', use_composition])),
         actions=[
+            SetParameter("use_sim_time", use_sim_time),
```

Comment:
I first expected to use `SetUseSimTime(True)` but that didn't work. Upon further inspection of the [Action](https://github.com/ros2/launch_ros/blob/master/launch_ros/launch_ros/actions/set_use_sim_time.py#L28), I don't think it's meant to set the parameter for all the nodes in the scope. I'm not really sure what it's for.

Reply:
Why not use this for each node then?

Reply:
I suppose just using the macro as Launch provides is nice, but if there's a strong reason to set it manually this way, that's OK too.